patches/resolve-package-path+4.0.3.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/node_modules/resolve-package-path/lib/index.js b/node_modules/resolve-package-path/lib/index.js
+index a1463f7..00d83b8 100644
+--- a/node_modules/resolve-package-path/lib/index.js
++++ b/node_modules/resolve-package-path/lib/index.js
```

Comment:
Sent the author a patch https://github.com/stefanpenner/resolve-package-path/issues/62

Reply:
Problem is the module is installed in the same directory as the project, which in our case it isn't - the module is in the binary.